infrastructure/setup_sigma_dao.sql

```diff
@@ -0,0 +1,45 @@
+-- #Note: Indexer should be running before executing the queries present in this file
+
+-- create a sigma_daos table if does not exit already.
+CREATE TABLE IF NOT EXISTS sigma_daos (
+  app_id bigint PRIMARY KEY, -- application id
+  app_params jsonb, -- application params with approval etc.
+  token_id bigint, -- token id
+  token_name varchar(20) -- token name
+);
+
+-- create a procedure to handle the trigger (update_app) action
+CREATE OR REPLACE FUNCTION update_app_info()
+RETURNS TRIGGER
+AS $$
+DECLARE
+    token_id bigint;
+	token_name varchar(20);
+	log_sigma text := 'U2lnbWFEQU8gY3JlYXRlZA==';
+	i record;
+BEGIN
+	FOR i IN SELECT txn FROM txn WHERE asset = NEW.asset AND typeenum=6 LOOP
+		IF i.txn::jsonb -> 'dt'->'lg' ?& ARRAY[log_sigma] THEN
+			-- store token id from db to above decalred variable token_id
+			SELECT index INTO token_id FROM public.asset ORDER BY index DESC LIMIT 1;
```

Comment:
the DAO token ID is not the most recent asset. It  can be any asset. You should read it from the app global state or a log.

Reply:
Yes, I know this. Currently, we are not storing asset id in a global state or log so it's hard to determine which asset was along with the app. This issue will be resolved once the asset has been moved to a global state with the hash code task. https://www.pivotaltracker.com/story/show/182178388

Reply:
please add a subtask there and TODO in the SQL code here

Reply:
Done